data/ecosystems/c/celo.toml

```diff
@@ -25,10 +25,112 @@ github_organizations = [
   "https://github.com/Ubeswap",
   "https://github.com/ReSource-Network",
   "https://github.com/Tradegen",
-  "https://github.com/centfinance"
+  "https://github.com/centfinance",
```

Comment:
cent.finance is also multi ecosystem.  A lot of multi network wallts wallets can either be sub ecosystems of General or just exist by themselves since it is hard to attribute wallet work to any one particular ecosystem.

Reply:
how should we best attribute these be multichain type projects?

Reply:
@jubos We invest in working with more comprehensive ecosystem partners that are multi-chain focus. How can we ensure contributions to these efforts are reflected in metrics for the end-of-year developer report with Celo?

Reply:
I think a "bridged" or "multichain" type is needed here in future, but with the option for identifying which chain is "home".